user/tests/wiring/no_fixture/cellular.cpp

```diff
@@ -157,6 +157,50 @@ test(CELLULAR_07_rssi_is_valid) {
     assertMoreOrEqual(s.rssi, -150);
 }
 
+test(CELLULAR_08_sigstr_is_valid) {     // collects signal strength and quality and validates RAT
+    connect_to_cloud(6*60*1000);
+    assertTrue(Particle.connected());
+    CellularSignal s;
+    // Verifies 10 times in a row that we are getting valid values for signal strength
+    for (int x = 0; x < 10; x++) {
+        s = Cellular.RSSI();
+        // Verify strength and quality values every single time
+        switch (s.getAccessTechnology()) {
+            case NET_ACCESS_TECHNOLOGY_GSM:     // GSM strength [-111, -48] and quality [-0.60, -3.70]
+                assertLessOrEqual(s.getStrengthValue(), -48.0f);
+                assertMoreOrEqual(s.getStrengthValue(), -111.0f);
+                assertLessOrEqual(s.getQualityValue(), -0.6f);
+                assertMoreOrEqual(s.getQualityValue(), -3.7f);
+                break;
+            case NET_ACCESS_TECHNOLOGY_EDGE:     // EDGE strength [-111, -48] and quality [-0.60, -3.70]
+                assertLessOrEqual(s.getStrengthValue(), -48.0f);
+                assertMoreOrEqual(s.getStrengthValue(), -111.0f);
+                assertLessOrEqual(s.getQualityValue(), -0.6f);
+                assertMoreOrEqual(s.getQualityValue(), -3.7f);
+                break;
+            case NET_ACCESS_TECHNOLOGY_UTRAN:     // UTRAN strength [-121, -25] and quality [-24.5, 0]
+                assertLessOrEqual(s.getStrengthValue(), -25.0f);
+                assertMoreOrEqual(s.getStrengthValue(), -121.0f);
+                assertLessOrEqual(s.getQualityValue(), 0.0f);
+                assertMoreOrEqual(s.getQualityValue(), -25.4f);
+                break;
+            case NET_ACCESS_TECHNOLOGY_LTE:
+            case NET_ACCESS_TECHNOLOGY_LTE_CAT_M1:  // LTE CAT-M1 strength [-141, -44] and quality [-20, -3]
+            case NET_ACCESS_TECHNOLOGY_LTE_CAT_NB1:
+                assertLessOrEqual(s.getStrengthValue(), -44.0f);
+                assertMoreOrEqual(s.getStrengthValue(), -141.0f);
+                assertLessOrEqual(s.getQualityValue(), -3.0f);
+                assertMoreOrEqual(s.getQualityValue(), -20.0f);
+                break;
+            default:
+                assertFalse((s.getAccessTechnology() == NET_ACCESS_TECHNOLOGY_UNKNOWN) \
+                        || (s.getAccessTechnology() == NET_ACCESS_TECHNOLOGY_NONE));
+                break;
+        }
+        delay(5000);
```

Comment:
Can we speed this up?  Maybe 500ms delay?  Also this one seems susceptible to start up issues.  Hopefully the previous test 07 delays enough that we have valid RSSI values.  The other RSSI test delays 5s between tries because it exits the test on the first valid response.  We could do a similar thing here (x = 10; break;).  Ideally since there are 130 - 150 individual tests for all platforms, we want to make each one as fast as possible to keep the testing process quick.